soc/arm/arm/musca_a/arm_mpu_regions.c

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright (c) 2018 Linaro Limited.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <soc.h>
+#include <arch/arm/cortex_m/mpu/arm_mpu.h>
+
+static const struct arm_mpu_region mpu_regions[] = {
+	/* Region 0 */
+	{ CONFIG_FLASH_BASE_ADDRESS,
+			 "FLASH_0",
+			 REGION_FLASH_ATTR(CONFIG_FLASH_BASE_ADDRESS,
+					   CONFIG_FLASH_SIZE*1024)},
+	/* Region 1 */
+	{CONFIG_SRAM_BASE_ADDRESS,
```

Comment:
minor: leave a space here, to align with line 12, or remove the space from line 12 :)

Reply:
Done, reworked formating